### PR TITLE
Fix protocol selection Helm values

### PR DESCRIPTION
**Pull request by nacx**
Fix the protocol selection page and use the new Helm values used by Pilot.

### CODE REVIEW

**Comment by ericvn** [comment]
See #6278, but should this text show that this is enabled for both inbound and outbound by default in 1.5?

File: content/en/docs/ops/configuration/traffic-management/protocol-selection/index.md [hunk 1]
```diff
@@ -52,4 +52,7 @@ spec:
 
 Istio can automatically detect HTTP and HTTP/2 traffic. If the protocol cannot automatically be determined, traffic will be treated as plain TCP traffic.
 
-This feature is experimental and off by default. It can be turned on by providing the install option `--set values.pilot.enableProtocolSniffing=true`.
+This feature is experimental and only enabled for outbound traffic by default. It can be turned on by providing the following install options:
```
Comment (yxue) [inline content/en/docs/ops/configuration/traffic-management/protocol-selection/index.md]: On master, both inbound and outbound are enabled by default.

**Comment by nacx** [comment]
Changed the text to indicate it is now enabled by default.

**Pull request merged**